src/test/frontend/replicasetlist/logsmenu_controller_test.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import LogsMenuController from 'replicasetlist/logsmenu_controller';
+import replicaSetListModule from 'replicasetlist/replicasetlist_module';
+
+describe('Logs menu controller', () => {
+  /**
+   * Logs menu controller.
+   * @type {!LogsMenuController}
+   */
+  let ctrl;
+
+  beforeEach(() => {
+    angular.mock.module(replicaSetListModule.name);
+
+    angular.mock.inject(($resource, $log) => { ctrl = new LogsMenuController($log, $resource); });
```

Comment:
Use `$controller` to instantiate controller. Take a look at other files for how to do this. With $controller tests are easier to maintain